RallyHereIntegration/Source/RallyHereIntegration/Private/RH_EntitlementHelpers.h

```diff
@@ -107,10 +107,25 @@ class FRH_EntitlementProcessor : public FRH_AsyncTaskHelper
 	 */
 	void QueryEntitlements()
 	{
-		if(PurchaseSubsystem != nullptr && !bIsOverride)
+		if(!bIsOverride)
 		{
-			PurchaseSubsystem->QueryReceipts(*OSS->GetIdentityInterface()->GetUniquePlayerId(LocalUserNum), false,
-			FOnQueryReceiptsComplete::CreateSP(this, &FRH_EntitlementProcessor::QueryEntitlementsComplete, OSS));
+			if (OSS.IsValid() && OSS->GetIdentityInterface().IsValid() && PurchaseSubsystem.IsValid())
+			{
+				auto UniquePlayerId = OSS->GetIdentityInterface()->GetUniquePlayerId(LocalUserNum);
```

Comment:
Not critical, but I'm strongly in favor of flattening these if checks out into early returns instead of nesting them

Reply:
Yeah, this isnt the nicest, was mostly trying to make sure the bases were covered.  Likely will do a second pass on this to clean things up now that I think I found all the potential problem spots.